src/main/java/seedu/address/logic/commands/AddPairCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ELDERLY;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_VOLUNTEER;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.pair.Pair;
+
+/**
+ * Adds a person to the address book.
+ */
+public class AddPairCommand extends Command {
+
+    public static final String COMMAND_WORD = "add_pair";
```

Comment:
```suggestion
    public static final String COMMAND_WORD = "add:pair";
```
on second thought, this syntax is more soothing to the eye 😄 what do the rest think?

Reply:
Lets discuss this in the next meeting